lineman/app/components/flash/flash.coffee

```diff
@@ -3,14 +3,17 @@ angular.module('loomioApp').directive 'flash', ->
   templateUrl: 'generated/components/flash/flash.html'
   replace: true
   controllerAs: 'flash'
-  controller: ($scope, $timeout, FlashService) ->
+  controller: ($scope, $interval, FlashService) ->
     $scope.pendingDismiss = null
 
     $scope.$on 'flashMessage', (event, flash) =>
       $scope.flash = _.merge flash, { visible: true }
       $timeout.cancel $scope.pendingDismiss if $scope.pendingDismiss?
-      $scope.pendingDismiss = $timeout $scope.dismiss, 2500
+      $scope.pendingDismiss = $interval($scope.dismiss, flash.duration, 1)
```

Comment:
This change is just to make protractor pass. This makes me sad.

https://stackoverflow.com/questions/25062748/testing-the-contents-of-a-temporary-element-with-protractor/25080187#25080187